Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM  debian:latest
+
+# Credit goes to https://github.com/headstar/sphinx-doc-docker
+
+RUN   apt-get update
+
+RUN   DEBIAN_FRONTEND=noninteractive apt-get install -y python-pip
+RUN   DEBIAN_FRONTEND=noninteractive apt-get install -y texlive texlive-latex-recommended texlive-latex-extra texlive-fonts-recommended
+RUN   DEBIAN_FRONTEND=noninteractive apt-get install -y enchant
+RUN   apt-get update --fix-missing
+RUN   DEBIAN_FRONTEND=noninteractive apt-get install -y git
+
+RUN   pip install Sphinx==1.4
```

Comment:
Can we install these in one command to create less layers? Maybe we can use the requirements.txt from the project to avoid duplication?